style.css

```diff
@@ -0,0 +1,67 @@
+html{
+    font-size: 15px;
+}
+img{
+    width: 50px;
+    height: 50px;
+}
+header{
+   text-align: center; 
+}
+
+#team_selector{
+    overflow-x: scroll;
+}
+
+.bg{
+    background-image: url("assets/photos/soccer-field.jpeg");
+    background-color:hsl(240,100%,20%);
+    height: 2000px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+}
+
+.date{
+   text-align: center; 
+}
+
+.match-time{
+    margin:auto;
+    text-align: center;
+}
+.navbar{
+    margin-bottom: 5px;
+}
+#score-start{
+    border-top: 2px solid black;
+}
+.scoreboard{
+    background-color: hsla(240,100%,80%, .7);;
+    border: 2px solid black;
+    margin:-2px;
+    width: 100%;
+}
+#score-team{
+    width: 40px;
+    height: 40px;
+}
+.home-team{
+    margin-bottom: 5px;
+    padding-top: 5px;
+    justify-content: center;
+    flex-basis: 100px;
+
+}
+.away-team{
+    margin-top: 5px;
+    padding-bottom: 5px;
+    flex-basis: 100px;
+}
+
+p{
```

Comment:
if we're overriding existing elements such as h1, h2, p, there is an order in the file in which we would do this, please look up said order and place the overrides in that order, it'll be easier to maintain